modules/rosa-sts-aws-requirements-association-concept.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa-sts-aws-prereqs.adoc
+// * rosa_planning/rosa-sts-ocm-role.adoc
+//
+:_content-type: CONCEPT
+= AWS account association
+
+{product-title} (ROSA) cluster-provisioning tasks require linking `ocm-role` and `user-role` {cluster-manager} IAM resources to your AWS account using your Amazon Resource Name (ARN).
+
+The `ocm-role` ARN is stored as a label in your Red Hat organization while the `user-role` ARN is stored as a label inside your Red Hat user account. Red Hat uses these ARN labels to confirm that the user is a valid account holder and that the correct permissions are available to perform the necessary tasks in the AWS account.
+
+.Prerequisites
+
+* You have an AWS account.
+* You are using {cluster-manager-url} to create clusters.
+* You have the permissions required to install AWS account-wide roles.
```

Comment:
This is not strictly true. We provide a 'manual' process in case the user does not have these permissions. They can send the output of this 'manual' process to someone within their organization that has the necessary access to complete the account-wide process.

Also, if the AWS account already has account-wide roles and they are just enabling the UI, then this prerequisite has been met without the user needing the permissions.

Reply:
Thanks @vkareh! I created this note instead:

![image](https://user-images.githubusercontent.com/16167833/173664831-3cf7c3a4-96ba-4342-a380-ef023571c2c4.png)

@arendej do you think this is useful to include?